src/hardware/mame/fmopl.cpp

```diff
@@ -1958,23 +1958,23 @@ static void OPL_clock_changed(FM_OPL *OPL, uint32_t clock, uint32_t rate)
 /* 'rate'  is sampling rate  */
 static FM_OPL *OPLCreate(device_t *device, uint32_t clock, uint32_t rate, int type)
 {
-	char *ptr;
-	FM_OPL *OPL;
-	int state_size;
-
 	if (FM_OPL::LockTable(device) == -1) return 0;
 
 	/* calculate OPL state size */
-	state_size  = sizeof(FM_OPL);
+	int state_size  = sizeof(FM_OPL);
```

Comment:
I suggest `const int` :)

Reply:
There's an #if just after this that increments `state_size` 
(unless I'm missing something!)  leaving unresolved until confirmed.